config/features.yml

```diff
@@ -191,10 +191,6 @@ features:
     actor_type: user
     description: Fast tracks 526 disability compensation hypertension claims by submitting additional health data
     enable_in_development: true
-  rrd_asthma_compensation:
-    actor_type: user
-    description: Fast tracks 526 disability compensation asthma claims by submitting additional health data
-    enable_in_development: true
```

Comment:
Moved down to alphabetize flipper names and keep RRD-related Flipper flags together